tardis/io/schemas/montecarlo.yml

```diff
@@ -17,7 +17,7 @@ properties:
   no_of_packets:
     type: number
     multipleOf: 1.0
-    description: Seed for the random number generator
+    description: This gives the number of packets normally used in each iteration.
```

Comment:
I don't fully agree with the word "normally" here. I would specify that it is used for all but the final iteration.

Reply:
Thanks for reviewing!
Will look into it for corrections...